code/controllers/subsystem/objects.dm

```diff
@@ -17,14 +17,12 @@ var/datum/subsystem/objects/SSobj
 
 /datum/subsystem/objects/Initialize(timeofday, zlevel)
 	setupGenetics()
-	for(var/atom/movable/AM in world)
-		if (zlevel && AM.z != zlevel)
+	for(var/atom/A in world)
+		if (zlevel && A.z != zlevel)
```

Comment:
nitpick, but isn't != higher priority than &&? this means we could save on one comparison for each atom if we put the != below the if(zlevel)
in fact, this would be much faster if it used block() instead of looping world, but the zlevel argument is unused right now either way, so it doesn't really matter